int32 camera_command
int32 RECORD = 0
int32 STOPRECORD = 1